spd/registry.py

```diff
@@ -81,13 +81,6 @@ class ExperimentConfig:
         expected_runtime=60,
         canonical_run=None,
     ),
-    # NOTE: This will be deprecated when we replicate runs with ss_llama_simple
-    "ss_llama": ExperimentConfig(
-        task_name="lm",
-        decomp_script=Path("spd/experiments/lm/lm_decomposition.py"),
-        config_path=Path("spd/experiments/lm/ss_llama_config.yaml"),
-        expected_runtime=2000,
-    ),
```

Comment:
I think Lucius still does ss_llama runs. I guess I'm OK with this failing hard and him having to manually put it in the registry in the future.

Fyi I added #92 which would handle this (not to be done in this PR though).